inspire_crawler/tasks.py

```diff
@@ -47,13 +47,48 @@
 
 
 @shared_task(ignore_results=True)
-def submit_results(job_id, results_uri, errors, log_file):
-    """Check results for current job."""
-    results_path = urlparse(results_uri).path
-    if not os.path.exists(results_path):
-        raise CrawlerInvalidResultsPath(
-            "Path specified in result does not exist: {0}".format(results_path)
+def submit_results(job_id, errors, log_file, results_uri, results_data=None):
+    """Check results for current job.
```

Comment:
It's called `submit_results` but then it actually reads `Check results...` What is actually doing? :)